script.js

```diff
@@ -25,3 +25,30 @@ if (hamburger && nav) {
 //             nav.style.display = 'block';
 //         }
 //     });
+
+// TECHSTACK SPAN
+
+const techStack = ['Python', 'Flask', 'Javascript', 'HTML', 'CSS'];
+const techStackSpan = document.getElementById('typewriter');
+const pauseBetweenWords = 1500;
+const typingInterval = 150;
+let indexTechStack = 0;
+let indexWord = 0;
+
+function typeTechStack() {
```

Comment:
I'd probably just add a comment here explaining how this works because guaranteed you'll forget in 3-6 months.